coupling/configurations/TimeIntegrationConfiguration.h

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2023 Helmut Schmidt University
+// This file is part of the Mamico project. For conditions of distribution
+// and use, please see the copyright notice in Mamico's main folder
+
+#pragma once
+
+#include "tarch/configuration/Configuration.h"
+#include "tarch/configuration/ParseConfiguration.h"
+#include <iostream>
+
+namespace coupling {
+namespace configurations {
+class TimeIntegrationConfiguration;
+}
+} // namespace coupling
+
+/** 
+ *  Reads time integration configuration. Use number-subdomains="1" to disable PinT. 
+ *  Derived from tarch::configuration::Configuration
+ *	@brief reads time integration configuration
+ *  @author Piet Jarmatz
+ */
+class coupling::configurations::TimeIntegrationConfiguration : public tarch::configuration::Configuration {
+public:
+  /** Constructor, initializes the class  */
+  TimeIntegrationConfiguration() : _pint_domains(1), _pint_iterations(1),
+  _auto_iteration(false), _visc_multiplier(1), _isValid(true) {}
+
+  /** Destructor */
+  virtual ~TimeIntegrationConfiguration() {}
+
+  void parseSubtag(tinyxml2::XMLElement* node) {
+    int buf;
+    tarch::configuration::ParseConfiguration::readIntMandatory(buf, node, "number-subdomains");
+    if (buf <= 0) {
+      std::cout << "ERROR coupling::TimeIntegrationConfiguration: "
+                   "number-subdomains = "
+                << buf << "!" << std::endl;
+      _isValid = false;
+      exit(EXIT_FAILURE);
+    }
+    _pint_domains = buf;
+
+    if(isPinTEnabled()){
+      #if (COUPLING_MD_PARALLEL != COUPLING_MD_YES)
```

Comment:
Maybe to be consistent, remove the tab in front of #if and #endif

Reply:
Okay, done, I just added TimeIntegrationConfiguration.h to CLANGFORMAT_SOURCES